app/lib/munkireport/AuthSaml.php

```diff
@@ -184,6 +184,12 @@ private function sls()
                 echo '<p>' . implode(', ', $errors) . '</p>';
             }
         } catch (OneLogin_Saml2_Error $e) {
+            if(isset($this->config['disable_sso_sls_verify']) && $this->config['disable_sso_sls_verify'] === true){
```

Comment:
Isn’t there a specific error coming from the framework that you could use?

Reply:
i will narrow this down